automation/upload_android_symbols.sh

```diff
@@ -48,5 +48,6 @@ for i in "${!TARGET_ARCHS[@]}"; do
 done
 
 # 2. Upload them.
+TOKEN_FILE='.token'
```

Comment:
Nit: I'd call this file with a less generic name to avoid collision and to be more explicit about the type of expected token. For instance `.symbols_upload_token.